toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -701,6 +706,11 @@ void FarmServer::queryHwInfo(HwInfo &hwInfo)
 	mib[1] = HW_MEMSIZE;
 	len = sizeof(physMemSize);
 	sysctl(mib, 2, &physMemSize, &len, NULL, 0);
+#endif
+
+#ifdef LINUX
+	TINT64 physMemSize = (TINT64)sysconf(_SC_PHYS_PAGES) * (TINT64)sysconf(_SC_PAGE_SIZE);
```

Comment:
I can’t point to the lines below, but look at the lines 719-721, you can provide a correct value for those, as I explained in https://github.com/opentoonz/opentoonz/pull/61#discussion_r57526961 in which case you can put them back to the OSX case only.